src/tree.h

```diff
@@ -159,7 +167,7 @@ union node_data {
 	struct if_call_data *ic_data;
 	struct declaration_data *d_data;
 	struct fc_entry *fc_data;
-	struct type_attribute_data *ta_data;
+	struct attribute_data *ta_data;
```

Comment:
This probably shouldn't be named "ta_data" anymore if the struct isn't named type_attribute_data anymore.

Reply:
changed to `at_data`